lib/realtime/tenants/authorization.ex

```diff
@@ -36,19 +36,13 @@ defmodule Realtime.Tenants.Authorization do
   * role: JWT role
   """
   @spec build_authorization_params(map()) :: t()
-  def build_authorization_params(%{
-        channel: channel,
-        headers: headers,
-        jwt: jwt,
-        claims: claims,
-        role: role
-      }) do
+  def build_authorization_params(map) do
     %__MODULE__{
-      channel: channel,
-      headers: headers,
-      jwt: jwt,
-      claims: claims,
-      role: role
+      channel: Map.get(map, :channel),
+      headers: Map.get(map, :headers),
+      jwt: Map.get(map, :jwt),
+      claims: Map.get(map, :claims),
+      role: Map.get(map, :role)
```

Comment:
Theoretically I prefer the previous code, as it is more assertive against typos and missing keys, although maybe these values can be missing now, hence the change? :)